samples/CMakeLists.txt

```diff
@@ -45,6 +45,8 @@ endif()
 
 add_executable(amber ${AMBER_SOURCES})
 target_include_directories(amber PRIVATE "${CMAKE_BINARY_DIR}")
+target_include_directories(amber PRIVATE
+  "${PROJECT_SOURCE_DIR}/third_party/lodepng")
```

Comment:
Change this to just ${PROJECT_SOURCE_DIR}/third_party

Reply:
done